src/dbos-executor.ts

```diff
@@ -423,8 +431,23 @@ export class DBOSExecutor {
 
     const runWorkflow = async () => {
       let result: R;
+      // Track concurrent executions of the WF
+      if (!this.workflowStats.has(wf.name)) {
+        this.workflowStats.set(wf.name, new WorkflowStats());
+      }
+      const wfs = this.workflowStats.get(wf.name)!;
+      wfs.currentlyExecuting++;
+      wfs.nStarted++;
+
       // Execute the workflow.
       try {
+        // If this is a scheduled workflow, do the pre-workflow things
```

Comment:
Maybe clarify what is the "pre-workflow things" if we want to comment =)

Reply:
How about pre-workflow work :-P

Reply:
🤦‍♂️

Reply:
Or "stuff" :-D